### PR TITLE
Weather kit and location integration

### PR DESCRIPTION
**Pull request by yukim17**

### CODE REVIEW

File: RayD/RayD/UI/MainView/MainViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                guard uvIndex != 0 else {
+                    self.noSunAlert = true
+                    self.state = .stop
+                    return
+                }
+                self.setupTimer(with: uvIndex)
+            })
+            .store(in: &subscriptions)
+
+        buttonTap
```
Comment (slavny11) [inline RayD/RayD/UI/MainView/MainViewModel.swift:64]: peredelyvai

**Review by slavny11 — CHANGES_REQUESTED** [review]
discussed already what to change

File: RayD/RayD/UI/MainView/MainViewModel.swift (hunk 1 above)
Comment (yukim17) [reply RayD/RayD/UI/MainView/MainViewModel.swift]: done

**Review by palant-dev — APPROVED** [review]
Interesting solutions I will discuss with you about some stuff later

**Pull request merged**